modules/nextflow/src/main/groovy/nextflow/Nextflow.groovy

```diff
@@ -57,6 +57,14 @@ class Nextflow {
 
     private static final Random random = new Random()
 
+    /**
+     * Get the value of an environment variable from the launch environment.
+     *
+     * @param name
```

Comment:
Let's improve your engineer karma having a better java doc 😄

Reply:
what do you mean

Reply:
Add a description for `name` and the method result

Reply:
Ever wrote javadoc 👉 3ca75fd7 ?

Reply:
seems redundant to me 🤷‍♂️

Reply:
This how javadoc is expected to be written 🤷‍♂️